packages/react-vapor/src/components/select/hoc/tests/MultiSelectWithFilter.spec.tsx

```diff
@@ -1,88 +1,73 @@
 import {MultiSelectWithFilter} from '../SelectComponents';
-import {ISelectWithFilterOwnProps} from '../SelectWithFilter';
 
 describe('Select', () => {
     describe('MultiSelectWithFilter', () => {
-        let wrapper: ReactWrapper<any, any>;
-        let store: Store<IReactVaporState>;
-
         const id: string = 'multi-select-with-filter';
 
-        const mountMultiSelect = (props: Partial<ISelectOwnProps & ISelectWithFilterOwnProps> = {items: []}) => {
-            wrapper = mount(
-                <Provider store={store}>
-                    <MultiSelectWithFilter id={id} {...props} />
-                </Provider>,
-                {attachTo: document.getElementById('App')}
-            );
-
-            store.dispatch(toggleSelect(id, true));
-            wrapper.update();
-        };
-
-        beforeEach(() => {
-            store = TestUtils.buildStore();
-        });
-
-        afterEach(() => {
-            store.dispatch(clearState());
-            wrapper?.unmount();
-        });
-
-        beforeAll(() => {
-            TestUtils.makeDebounceStatic();
-        });
-
         it('should add a duplicate if the filterValue is already selected', () => {
             const filterValue: string = 'a';
             const duplicateText: string = 'duplicate';
 
-            mountMultiSelect({items: [{value: filterValue, selected: true}], duplicateText});
-            store.dispatch(filterThrough(id, filterValue));
+            render(
+                <MultiSelectWithFilter
+                    id={id}
+                    duplicateText={duplicateText}
+                    items={[
+                        {value: filterValue, selected: true},
+                        {value: filterValue, selected: true},
+                    ]}
+                />,
+                {
+                    initialState: {},
+                }
+            );
 
-            wrapper.update();
-            const itemBox = wrapper.find(SelectConnected).find(ItemBox).first();
+            const listItems = screen.getAllByRole('listitem');
 
-            expect(itemBox.props().value).toBe(duplicateText);
+            expect(listItems.length).toBe(2);
+            expect(listItems[0]).toHaveTextContent(listItems[1].textContent);
         });
 
         it('should open the dropdown even if the list is empty with customValue', () => {
             const noItemsText = 'not an item text';
 
-            mountMultiSelect({items: [], noItemsText, customValues: true});
-
-            const itemBox = wrapper.find(SelectConnected).find(ItemBox).first();
+            render(<MultiSelectWithFilter id={id} noItemsText={noItemsText} customValues items={[]} />, {
+                initialState: {},
+            });
 
-            expect(itemBox.props().value).toBe(noItemsText);
+            expect(
+                screen.getByRole('option', {
+                    hidden: true,
+                })
+            ).toHaveTextContent(noItemsText);

```

Comment:
From the test name you should open the dropdown before validating that the option is there. Then you wouldn't need the `hidden: true`